src/scripts/globalize.js

```diff
@@ -68,6 +68,9 @@ define(['connectionManager', 'userSettings', 'events'], function (connectionMana
                 ensureTranslation(allTranslations[i], fallbackCulture);
             }
         }
+        if (culture == 'zh-tw' || culture == 'zh-hk') {
+            require(["css!jellyfin-noto/css/TC"]);
+        }
```

Comment:
What about other CJK variants? At least SC?

Reply:
Loaded by default

Reply:
I think Japanese should be loaded by default (Because of Hiragana and Katakana). Korean as well, since the Korean font includes only Hangul.

Both variants of Chinese should be loaded only on their respective cultures for, as we're not really sure what the usages are compared to each other and to Japanese.

Reply:
@MrTimscampi That's the current situation. All of them are loaded by jellyfin-noto package (also TC before, I thought it used other ranges, so I imported all of them at once). Now TC is not imported (only when required) and SC is used as a fallback for all the other languages (so users with other languages will see characters in SC by default).

The behaviour of the other charsets are unchanged, so everything it's still the same in that matter.

Reply:
No, what I mean is that Japanese and TC/SC share characters, but with different variations :)

So, without being sure, we should load the Chinese fonts only if explicitly required by the user. We don't know if Simplified Chinese replaces characters from Japanese with versions that don't look native to Japanese people. But as Japanese has characters that Chinese doesn't have (Hiragana and Katakana), it should be loaded by default (like now).

Neither Simplified Chinese nor Traditional Chinese should be loaded by default. These should be applied only if the user has selected their respective languages.